arangod/Aql/ExecutionBlockImpl.cpp

```diff
@@ -229,7 +229,9 @@ struct ExecuteSkipVariant<SkipVariants::FETCHER> {
   static std::tuple<ExecutionState, typename Executor::Stats, size_t> executeSkip(
       Executor& executor, typename Executor::Fetcher& fetcher, size_t toSkip) {
     auto res = fetcher.skipRows(toSkip);
-    return {res.first, typename Executor::Stats{}, res.second};
+    return std::make_tuple<ExecutionState, typename Executor::Stats, size_t>( // tupple, cannot use initializer list due to build failure
+      std::move(ExecutionState(res.first)), typename Executor::Stats{}, size_t(res.second) // rvalue required for all elements to match constructor signature
```

Comment:
```suggestion
      ExecutionState(res.first), typename Executor::Stats{}, size_t(res.second) // rvalue required for all elements to match constructor signature
```

Reply:
I would think so too, but MSVC was complaining with:
cannot convert argument 1 from 'arangodb::aql::ExecutionState' to 'arangodb::aql::ExecutionState &&'